draft-ietf-taps-interface.md

```diff
@@ -2388,6 +2408,33 @@ The Capacity Profile for a selected protocol stack may be modified on a
 per-Message basis using the Transmission Profile Message Property; see
 {{send-profile}}.
 
+### Ojective of Using Multiple Paths {#multipath-objective}
+
+Name:
+: multipath-objective
+
+Type:
+: Enumeration
+
+Default:
+: Handover
+
+This property specifies the local objective of transferring data across multiple paths between the same end hosts if Parallel Use of Multiple Paths not set to Disabled (see {{multipath-mode}}). Possible values are:
+
+None:
+: Do not use other paths as long as the initial path is available, but allow the peer endpoint to make use it to achieve its objective.
```

Comment:
Does this value make sense? It seems like it's a duplicate of Passive; why not just say Passive doesn't use the multi path policy?

Reply:
This property only effects the packet scheduler, not to the path manager as above. 
As a server, you may choose passive above and "Aggregate" or "Interactive" here.

I am unsure whether we need it, as the difference i see between "None" and "Handover" is really small. In iOS, I would expect it turns off WiFi Assist and does hand-over once a path breaks.

Reply:
I would remove this and only have "handover"